### ISSUE TITLE
RUCSS compatibility with Cost Calculator

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#5584]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.12.3.1`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
When RUCSS is used with the [Cost Calculator](https://codecanyon.net/item/cost-calculator-wordpress-plugin/12778927) plugin, the calculator form fields are missing most of their style. 

Safelisting didn't work. 

Preserving the following inline style: 

```
btcc_style_inline-inline-css
```

And the following CSS file: 

```
/bt_cost_calculator/style.min.css
```

fixed the issue. 

This doesn't require a compatibility file, as the file names a unique. 

But we don't have a way to preserve links to send them as a part of the dynamic lists in the backend. 

Putting this on hold until then.

**To Reproduce**
1. Use the Cost Calculator
2. Enable RUCSS
3. See error

**Expected behavior**
The calculator should render without any issues.

**Screenshots**
N/A

**Additional context**
Ticket - https://secure.helpscout.net/conversation/2068954161/381871?folderId=3864735
Slack - https://wp-media.slack.com/archives/C029G1B5HD2/p1668860006905199

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by piotrbak** [comment]
Blocked by:
https://github.com/wp-media/nodejs-treeshaker/issues/276
